libecl/src/ecl_grid.c

```diff
@@ -1066,6 +1066,21 @@ static void ecl_cell_taint_cell( ecl_cell_type * cell ) {
 }
 
 
+static int ecl_cell_get_twist( const ecl_cell_type * cell ) {
+  double sign = 1.0;
+  int twist_count = 0;
+
+  for (int c = 0; c < 4; c++) {
+    const point_type * p1 = &cell->corner_list[c];
+    const point_type * p2 = &cell->corner_list[c + 4];
+    if (sign * (p2->z - p1->z) < 0)
+      twist_count += 1;
+  }
+  return twist_count;
+}
```

Comment:
Aren't you going to use `sign`?  As it is now, the `if` statement is essentially `if (p1->z  > p2->z) twist_count++`.